lynxkite-graph-analytics/src/lynxkite_graph_analytics/core.py

```diff
@@ -42,17 +42,28 @@
 rendered as a dropdown in the frontend, listing the keys on the "other" part of the Bundle.
 The key is passed to the operation as a string."""
 
-ModelName = typing.Annotated[
+PyTorchModelName = typing.Annotated[
     str,
     {
         "format": "dropdown",
-        "metadata_query": "[].other.*[] | [?type == 'model'].key",
+        "metadata_query": "[].other.*[] | [?type == 'pytorch-model'].key",
     },
 ]
 """A type annotation to be used for parameters of an operation. ModelName is
 rendered as a dropdown in the frontend, listing the models in the Bundle.
 The model name is passed to the operation as a string."""
 
+PyKEENModelName = typing.Annotated[
```

Comment:
Please move this to the PyKEEN code file. (You could also move PyTorchModelName to the PyTorch code.)